src/frontend/Sider/Behandling/BehandlingTabsInnhold.tsx

```diff
@@ -64,47 +65,52 @@ const BehandlingTabsInnhold = () => {
 
     const behandlingFaner = hentBehandlingfaner(behandling.stønadstype);
     return (
-        <Tabs value={aktivFane} onChange={(e) => håndterFaneBytte(e as FanePath)}>
-            <StickyTablistContainer>
-                <TabsList>
-                    {behandlingFaner.map((tab) =>
-                        faneErLåst(tab.path) ? (
-                            <DisabledTab
-                                key={tab.path}
-                                value={tab.path}
-                                label={tab.navn}
-                                icon={tab.ikon}
-                            />
-                        ) : (
-                            <Tabs.Tab
-                                key={tab.path}
-                                value={tab.path}
-                                label={tab.navn}
-                                icon={tab.ikon}
-                            />
-                        )
-                    )}
-                    {behandlingErRedigerbar && !statusPåVentRedigering && (
-                        <Tabsknapp>
-                            <Button size={'small'} onClick={() => settStatusPåVentRedigering(true)}>
-                                Sett på vent
-                            </Button>
-                        </Tabsknapp>
-                    )}
-                </TabsList>
-            </StickyTablistContainer>
-
-            <SettPåVentContainer
-                statusPåVentRedigering={statusPåVentRedigering}
-                settStatusPåVentRedigering={settStatusPåVentRedigering}
-            />
-
-            {behandlingFaner.map((tab) => (
-                <Tabs.Panel key={tab.path} value={tab.path}>
-                    {tab.komponent(behandling.id)}
-                </Tabs.Panel>
-            ))}
-        </Tabs>
+        <StegProvider fane={isFanePath(aktivFane) ? aktivFane : undefined} behandling={behandling}>
```

Comment:
Hvorfor trengs `isFanePath(aktivFane)` ? 
`aktivFane` er alltid satt til en av fanene, fra
`const [aktivFane, settAktivFane] = useState<string>(path || FanePath.INNGANGSVILKÅR);`

Reply:
Måtte enten det, eller legge til caste `path` til `FanePath`. Og det føltes mindre hacky

Reply:
En annen mulighet er å flyttet typeguarden til `aktivFane`-staten. Da får vi litt bedre typing på den staten også:

```
    const [aktivFane, settAktivFane] = useState<FanePath>(
        isFanePath(path) ? path : FanePath.INNGANGSVILKÅR
    );
```

forøvrig tror jeg vi kan fjerne hele
```
    useEffect(() => {
        settAktivFane(path || FanePath.INNGANGSVILKÅR);
    }, [path]);
```

Reply:
Forøvrig tror jeg vi kan fjerne hele
```
    useEffect(() => {
        settAktivFane(path || FanePath.INNGANGSVILKÅR);
    }, [path]);
```

Reply:
Vi trenger den for å navigere til neste fane når man ferdigstiler et steg.